app-crypt/gcr/gcr-3.92.0.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit gnome.org gnome2-utils meson python-any-r1 vala xdg
+
+DESCRIPTION="Libraries for cryptographic UIs and accessing PKCS#11 modules"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gcr"
+
+LICENSE="GPL-2+ LGPL-2+"
+SLOT="0/1" # subslot = suffix of libgcr-base-3 and co
```

Comment:
We'll have to do something different here.

Reply:
We have one file collision, can't make it a slot easily therefore. 
However, I think this is a oversight by upstream, since all the other files are namespaced. 

https://gitlab.gnome.org/GNOME/gcr/-/issues/105

Reply:
Thanks for filing that issue.

Looks like as it is today, the only files that both 3.41.1 and 3.92 install are

````
 ./usr/libexec/gcr-ssh-agent
 ./usr/lib/systemd/user/gcr-ssh-agent.service
 ./usr/lib/systemd/user/gcr-ssh-agent.socket
````

Reply:
Looks like Fedora just removes these files from the older version of the package:

https://src.fedoraproject.org/rpms/gcr3/blob/rawhide/f/gcr3.spec#_78